### DISCUSSION TITLE
Calling a Go function with JSON string triggers a file load... why that?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [wailsapp/wails#857] [Q&A]
I have an HTML form where the submit function collects all inputs into a JSON string which I use as a parameter to my Go function. As soon as the Go function gets called, the console shows:

```
 [DesktopAssetServer] Loading file from disk: /?user_company=company&user_division=divisi...
```
Which of course fails because such a file doesn't exist. Why is this triggering a file load at all?

**Comment by Robert-M-Muench** [comment]
Here is the JS console output:

![image](https://user-images.githubusercontent.com/1826391/136090251-da2b44d5-f836-4890-b019-8a33421b0c76.png)

**Comment by leaanthony** [comment]
Please open a ticket with a reproducable example. Thanks!

**Reply by Robert-M-Muench** [reply]
Do you want to get a zip file with all necessary files? Or what is the best way to provide the reproducible example? Because there are many files involved.

**Reply by leaanthony** [reply]
Maybe just the Go method you are calling and the frontend payload to it.
